qemu/tests/cfg/blockdev_full_backup.cfg

```diff
@@ -77,7 +77,7 @@
                     auto-finalize = true
                 - manual_completed:
                     auto-dismiss = false
-                    auto-finalize = false
+                    auto-finalize = true
```

Comment:
hi @aliang123 , it looks that the test scenario changes if we set auto-finalize to true, the 'auto-finalize=false' test scenario was gone, is that OK?

Reply:
talked with aihua, backup will not cover auto-finalize=false, but other block jobs will cover this.